buildSrc/src/main/java/Dependencies.kt

```diff
@@ -26,6 +26,7 @@ private object Versions {
     const val androidx_lifecycle_savedstate = "1.0.0-alpha01"
     const val androidx_testing = "1.1.0-alpha08"
     const val androidx_core = "1.2.0-alpha01"
+    const val androidx_paging = "2.0.0"
```

Comment:
Newest one is `2.1.0` but I used `2.0.0` here because it was version used from transitive dependency.